.changeset/chatty-carrots-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
```

Comment:
This is an intentional, user-facing issue, so it probably should be more than a patch. A patch would be something like documentation change or a bugfix (correcting an unintentional, user-facing issue).

Reply:
Will update to be minor.

Reply:
Updated to minor. Resolving :)